app/controllers/ai_annotations_controller.rb

```diff
@@ -7,7 +9,9 @@ def create
     text = ai_annotation_params[:text]
     prompt = ai_annotation_params[:prompt]
     @new_ai_annotation = AiAnnotation.prepare_with(text, prompt)
-    ai_annotation = @new_ai_annotation.annotate!
+
+    ai_annotation, token_used = @new_ai_annotation.annotate!
+    increment_daily_token_usage(token_used)
```

Comment:
```suggestion
    ai_annotation = @new_ai_annotation.annotate!
    increment_daily_token_usage(@new_ai_annotation)
```

としても良さそうです。

increment_daily_token_usageメソッドの中でai_annotationからトークン使用数を取得するのをイメージしています。

AiAnnotationがトークン使用数を状態に持つのは違和感がありません。

ああ、でもConcernがモデルのAPIを知っているのは少し違和感があります。

```suggestion
    ai_annotation = @new_ai_annotation.annotate!
    increment_daily_token_usage(@new_ai_annotation.token_used)
```
とすると良さそうです。

Reply:
修正しました。
https://github.com/jdkim/textae_campus/pull/2/commits/4a994f2f58c0c6441730966d5bfa6721c5b40cb7
https://github.com/jdkim/textae_campus/pull/2/commits/de8b70dc2bed4dfbc62515a6be310b6510f7ac43